aviary/models/propellers/KingAir90.prop

```diff
@@ -0,0 +1,437 @@
+# created 06/18/24 at 12:09 by xjiang
```

Comment:
```suggestion
# created 06/18/24 at 12:09
```
I typically remove usernames as well for repo files, but not as important

Reply:
Done